backend/drive/drive.go

```diff
@@ -1001,10 +1007,18 @@ func (f *Fs) newBaseObject(remote string, info *drive.File) baseObject {
 
 // newRegularObject creates a fs.Object for a normal drive.File
 func (f *Fs) newRegularObject(remote string, info *drive.File) fs.Object {
+	// For Readability 
+	// - if SkipChecksumGphotos and file is type Photo or Video then skip.
+	doMd5sum := true
+	if f.opt.SkipChecksumGphotos && 
+		(info.MimeType == "application/vnd.google-apps.photo" || info.MimeType == "application/vnd.google-apps.video") {
+			doMd5sum = false
+	}
+	
 	return &Object{
 		baseObject: f.newBaseObject(remote, info),
 		url:        fmt.Sprintf("%sfiles/%s?alt=media", f.svc.BasePath, info.Id),
-		md5sum:     strings.ToLower(info.Md5Checksum),
+		md5sum:     doMd5sum && strings.ToLower(info.Md5Checksum),
```

Comment:
This didn't compile with

```
backend/drive/drive.go:1021:24: invalid operation: doMd5sum && strings.ToLower(info.Md5Checksum) (mismatched types bool and string)
```

You can use short-circuiting booleans in go unfortunately - go doesn't do any implicit type conversions.

I'd probably write something like

```go
        Md5sum := strings.ToLower(info.Md5Checksum)
	if f.opt.SkipChecksumGphotos && 
		(info.MimeType == "application/vnd.google-apps.photo" || info.MimeType == "application/vnd.google-apps.video") {
			Md5sum = ""
	}
```

Then use `md5sum: Md5Sum` a bit later.